client/signup/processing-screen/index.jsx

```diff
@@ -3,24 +3,51 @@
  */
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { find } from 'lodash';
+import { find, isEmpty } from 'lodash';
 import { localize } from 'i18n-calypso';
 import Gridicon from 'gridicons';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Button from 'components/button';
 import Notice from 'components/notice';
+import analytics from 'lib/analytics';
+import { abtest } from 'lib/abtest';
 
 export class SignupProcessingScreen extends Component {
 	static propTypes = {
 		hasCartItems: PropTypes.bool.isRequired,
 		loginHandler: PropTypes.func,
 		steps: PropTypes.array.isRequired,
 		user: PropTypes.object,
+		signupProgress: PropTypes.array,
 	};
 
+	componentWillMount() {
+		this.state = {
```

Comment:
@taggon 👋 This line is causing a warning in the console:
```
index.jsx:97 Warning: SignupProcessingScreen.componentWillMount(): Assigning directly to this.state is deprecated (except inside a component's constructor). Use setState instead.
```
Could you change it to `setState` instead?